mkdocs.yml

```diff
@@ -99,6 +100,7 @@ nav:
         - A&E Forecasting Tool: our_work/a_and_e_forecasting_tool.md
         - Cancer high-risk cohorts: our_work/cancer_high_risk_cohorts.md
         - Risk stratification models for Population and Person Insights (PaPI): our_work/Papi_Webpage.md
+        - CVD Pathways: our_work/cvd_pathways.md
```

Comment:
does CVD not fall more under the Data Science for Linked/Longitudinal Data category?